did-configuration.json

```diff
@@ -35,7 +35,7 @@
       "type": ["VerifiableCredential", "DomainLinkageCredential"],
       "credentialSubject": {
         "id": "did:web:vc.transmute.world",
-        "domain": "identity.foundation"
+        "origin": "https://identity.foundation"
```

Comment:
the fact that this property has changed, but the JWS has not is proof that the example has not been updated, and that the demo has been broken.